service/matching/matching_engine.go

```diff
@@ -2060,23 +2060,28 @@ func (e *matchingEngineImpl) updateTaskQueuePartitionGauge(pm taskQueuePartition
 		}
 	}
 
+	nsName := pm.Namespace().Name().String()
+
 	e.metricsHandler.Gauge(metrics.LoadedTaskQueueFamilyGauge.Name()).Record(
 		float64(loadedTaskQueueFamilyCounter),
-		metrics.NamespaceTag(pm.Namespace().Name().String()),
+		metrics.NamespaceTag(nsName),
 	)
 
 	metrics.LoadedTaskQueueGauge.With(e.metricsHandler).Record(
 		float64(loadedTaskQueueCounter),
-		metrics.NamespaceTag(pm.Namespace().Name().String()),
-		metrics.TaskTypeTag(taskQueueParameters.taskType.String()),
+		metrics.NamespaceTag(nsName),
+		metrics.TaskQueueTypeTag(taskQueueParameters.taskType),
 	)
 
-	metrics.LoadedTaskQueuePartitionGauge.With(e.metricsHandler).Record(
-		float64(loadedTaskQueuePartitionCounter),
-		metrics.NamespaceTag(pm.Namespace().Name().String()),
-		metrics.TaskTypeTag(taskQueueParameters.taskType.String()),
-		metrics.PartitionTypeTag(taskQueuePartitionParameters.partitionType.String()),
+	taggedHandler := metrics.GetPerTaskQueuePartitionScope(
+		e.metricsHandler,
+		nsName,
+		pm.Partition(),
+		// TODO: Track counters per TQ name so we can honor pm.config.BreakdownMetricsByTaskQueue(),
```

Comment:
is this useful? they'd be 0 or 1 only?

oh wait.. for loaded partitions, it'd be 0 or the partition count, for loaded task queue (family), it'd be 0 or 1?

Reply:
this is only for loaded partitions. the use would be you can see how many partitions are loaded for a tq. but not super important I guess.